.drone.yml

```diff
@@ -123,6 +123,21 @@ steps:
     depends_on:
       - git tag push
 
+  - name: url check
+    pull: if-not-exists    
+    image: alpine/curl:3.14   
+    commands:       
+    - curl -s https://raw.githubusercontent.com/UKHomeOffice/callisto-ui-nginx/main/scripts/is-url-up.sh >up.sh
+    - chmod +x ./up.sh
+    - ./up.sh https://web.dev.callisto-notprod.homeoffice.gov.uk/index.html
+    when:
+      event:
+       - push
+      branch:
+       - main  
+    depends_on:
+      - upload to S3    
+
```

Comment:
In the `callisto-ui-test-deploy` pipeline below isn't the deployment to test meant to happen automatically now eg. directly after deploying to dev. And the manual promotion only for prod..  Or was that to come in a separate PR?

Reply:
The manual promote to test step was added for the previous ticket 'Create manual promote to test'. 

My understanding for this ticket 'Create manual promote to Prod' was promoting to Prod would also be another separate manual promotion step.

Are you saying the promote to Test step should be merged into the deploy to Dev?

Reply:
Resolving after the team deployment discussion - concluded that we can decide the best strategy for the specific service.